datafusion/core/src/datasource/file_format/avro.rs

```diff
@@ -48,7 +48,7 @@ impl FileFormat for AvroFormat {
 
     async fn infer_schema(
         &self,
-        _state: &SessionState,
+        _task_ctx: &TaskContext,
```

Comment:
The point of this PR is to change the `SessionState` references in these traits to `TaskContext`